### ISSUE TITLE
postal make-user noninteractive

### ISSUE THREAD

**Issue by lukasmrtvy** [postalhq/postal#671]
Is possible to run `postal make-user` in noninteractive mode? 
I know I can write except script, but it would be much better to pass arguments.
Thanks

**Comment by lukasmrtvy** [comment]
Duplicate: https://github.com/atech/postal/issues/145
But without any status..

**Comment by willpower232** [comment]
I don't think that ever made it into the real world as the issue is still open as you say :-(

**Comment by zeeshanjamal16** [comment]
You can use bash feature and pipe the standard input to directly create user.

`printf 'test@example.com\nFname\nSname\nLongRand0mPassword' | postal make-user`

**Comment by stevus** [comment]
Can this be reopened?

```
Postal User Creator
Enter the information required to create a new Postal user.
This tool is usually only used to create your initial admin user.

E-Mail Address      : First Name          : Last Name           : Initial Password:   : /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/terminal/io_console.rb:20:in `echo=': Inappropriate ioctl for device (Errno::ENOTTY)
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/terminal/io_console.rb:20:in `restore_mode'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/terminal.rb:70:in `ensure in raw_no_echo_mode_exec'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/terminal.rb:70:in `raw_no_echo_mode_exec'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline.rb:538:in `get_line_raw_no_echo_mode'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline.rb:518:in `get_response_line_mode'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/question.rb:524:in `get_response'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/question.rb:536:in `get_response_or_default'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline/question_asker.rb:30:in `ask_once'
	from /opt/postal/vendor/bundle/ruby/2.4.0/gems/highline-2.0.0/lib/highline.rb:223:in `ask'
	from /opt/postal/lib/postal/user_creator.rb:16:in `start'
	from script/make_user.rb:7:in `<main>'
```

**Comment by willpower232** [comment]
I do not believe the feature has been implemented yet as per #145.
